app/src/main/res/layout/activity_bmi_calculator.xml

```diff
@@ -186,5 +191,35 @@
 
     </androidx.cardview.widget.CardView>
 
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="horizontal">
+
+        <ImageButton
+            android:id="@+id/btn_nutritionpage"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:contentDescription="@string/btnSpoon"
+            app:srcCompat="@drawable/spoon" />
+
+        <ImageButton
+            android:id="@+id/btn_caloriepage"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:contentDescription="@string/btnFire"
+            app:srcCompat="@drawable/fire" />
+
+        <ImageButton
+            android:id="@+id/btn_healthpage"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:contentDescription="@string/btnMore"
+            app:srcCompat="@drawable/add" />
+    </LinearLayout>
+
```

Comment:
Added the image buttons instead of the text buttons. You can also add the images you like after.
Sources:
https://www.youtube.com/watch?v=vXjJyLpydxg
https://www.youtube.com/watch?v=YaWmNhlOXYk

Reply:
In Android, Pictures are faster than words.  Use image buttons instead of text on buttons, where possible.